code/modules/clothing/shoes/jobs.dm

```diff
@@ -9,6 +9,8 @@
 	species_restricted = null
 	drop_sound = 'sound/items/drop/rubber.ogg'
 	pickup_sound = 'sound/items/pickup/rubber.ogg'
+	sprite_sheets = list(
+		"Tajara" = 'icons/mob/species/tajaran/shoes.dmi')
```

Comment:
can you put all of these on one line, splitting them like this is kinda disgusting

Reply:
ok king